source/dubregistry/registry.d

```diff
@@ -213,6 +213,21 @@ class DubRegistry {
 		return getPackageInfo(pack, include_errors);
 	}
 
+	/** Gets information about multiple packages at once.
+
+		The order of the returned packages is undefined and some of the package
+		names may not result in a `PackageInfo` to be returned at all, if no
+		package with the given name exists, so that the returned range needs
```

Comment:
s/so that/so

Reply:
I think the "so that" is okay here, but the sentence is borderline in length anyway, so I'll reword that.